notebooks/en/quantize_stable_diffusion_with_quanto.ipynb

```diff
@@ -0,0 +1,777 @@
+{
```

Comment:
<p>I think you can very briefly explain the important steps of the process in a paragraph instead of putting like this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>